.github/actions/setup-env/action.yml

```diff
@@ -1,27 +0,0 @@
-name: Setup Environment
-description: Does minimal setup of environment for a job
-
-runs:
-  using: composite
-  steps:
-    - name: Add extra required branches
-      if: ${{ github.event_name == 'pull_request' && github.event.pull_request.base.ref != 'main' }}
-      run: git fetch origin main:main ${{ github.event.pull_request.base.ref }}:${{ github.event.pull_request.base.ref }}
-      shell: bash
```

Comment:
I think this is still needed because otherwise, PRs targeting a branch different than `main` won't run

Reply:
We found that code is never triggered.